compiler/noirc_frontend/src/lexer/lexer.rs

```diff
@@ -1363,4 +1363,25 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn test_ensures_attribute() {
```

Comment:
That's great! Please, add a test for requires as well.

Reply:
Done